### ISSUE TITLE
ERROR:  current transaction is aborted, commands ignored until end of transaction block

### ISSUE THREAD

**Issue by beefo** [neisbut/Npgsql.Bulk#7]
I am getting error `ERROR:  current transaction is aborted, commands ignored until end of transaction block` on command `STATEMENT:  INSERT INTO "foobar" ("Foo", "DateTime", "Baz", "Bar") SELECT foobar_foo, foobar_datetime, foobar_baz, foobar_bar FROM _temp_636645699748660653 RETURNING Id`

```
var uploader = new NpgsqlBulkUploader(context);
var data = value.Bar.Select(u => new Fobar{ ... }).ToList();
uploader.Insert(data);
```

Any idea what I am doing wrong?

**Comment by israelleon** [comment]
hi, i get the same error. in my case, i am using Npgsql 4.0.0 and i am solving this error reinstalling Npgsql 3.2.7, reading the documentation the version 4 has changed the API import (COPY) .  reading more about this in http://www.npgsql.org/doc/release-notes/4.0.html.

sorry for my english. u.u.

**Comment by neisbut** [comment]
Hi @beefo, sorry for a late answer. @israelleon is right. There was a breaking change in Npgsql related to COPY. I've finally fixed this issue today, you can try latest 0.6.0 release. Thanks.
